tests/functional/WebDriverTestCase.php

```diff
@@ -213,4 +213,20 @@ protected function setUpSauceLabs()
             }
         }
     }
+
+    /**
+     * Uses assertStringContainsString when it is available or uses assertContains for old phpunit versions
+     * @param string $needle
+     * @param string $haystack
+     * @param string $message
+     */
+    protected function compatAssertStringContainsString($needle, $haystack, $message = '')
```

Comment:
Just in idea, isn't possible to name it simply "assertStringContainsString", and call the parent if it exists, or assertContains if not?

Once we drop old PHPUnit versions, we could then only remove this polyfill.

Reply:
@williamdes can you maybe check in on this one?

Reply:
No I can not do that, neither.
Because if I rename to `compatAssertStringContainsString` I will have to be compatible on the PHP signature, that would be one particular phpunit version. And so I would screw all the other ones.
Some not supporting phptype hints, some having some typehints defined, some no typehints defined.
:man_shrugging:

Reply:
Though the signature removal is already being done in via `sed` for similar cases like `setUpBeforeClass` etc., so theoretically it could be possible for `assertStringContainsString` signature as well, no? 

However it was just an idea, if you think its not worth it, we can keep it like this :).

Reply:
Well it is possible but totally overkill, let's keep it as it is ;)
I say it is overkill because we would need to make different seds for different phpunit versions and this is more complicated than just switching `: void` return typehints.
So is this okay for you until we drop some phpunit versions ?